src/test/java/com/okta/tools/aws/settings/ConfigurationTest.java

```diff
@@ -59,43 +56,30 @@
                     null, null, null, null, null,
                     0, null, "custom ", null);
 
-    /*
-     * Test instantiating a Credentials object with invalid INI.
-     */
-    @Test
-    void instantiateInvalidConfiguration() {
-        assertThrows(IOException.class, () -> new Configuration(new StringReader("someinvalidini")));
-    }
-
```

Comment:
This is technically a valid INI file. I couldn't figure out how to construct an invalid INI file. Ini4j got this wrong.